coffeescripts/cards-es.coffee

```diff
@@ -856,6 +856,9 @@ exportObj.cardLoaders['Español'] = () ->
         "Ion Torpedoes":
             name: "Torpedos de Iones"
             text: """<strong>Ataque (Blanco Fijado):</strong> Gasta tu ficha de Blanco Fijado y descarta esta carta para efectuar este ataque.<br /><br />Si este ataque impacta, el defensor y toda nave que esté a alcance 1 reciben 1 ficha de Iones cada una."""
+        "Bomb Loadout":
+            name: "Compartimento de Bombas"
```

Comment:
FYI: to fix the missing title you need to add

```coffeescript
ship: 'Ala-Y'
```

I'll fix in on my end after I merge.